src/angularfire2.spec.ts

```diff
@@ -15,80 +15,83 @@ import {
   AngularFire,
   FirebaseObjectObservable,
   FIREBASE_PROVIDERS,
-  FirebaseAuth,
-  FirebaseUrl,
-  FirebaseRef,
+  AngularFireAuth,
+  FirebaseConfig,
+  FirebaseApp,
   defaultFirebase,
-  FirebaseDatabase
+  FirebaseDatabase,
+  FirebaseAppConfig
 } from './angularfire2';
 import {Subscription} from 'rxjs';
 import 'rxjs/add/operator/toPromise';
 import 'rxjs/add/operator/take';
 import 'rxjs/add/operator/do';
 import 'rxjs/add/operator/delay';
 
-// Only use this URL for angularfire2.spec.ts
-const localServerUrl = 'https://angularfire2.firebaseio-demo.com/';
+export const firebaseConfig: FirebaseAppConfig  = {
```

Comment:
Move into single shared config file. There's one in test/config.ts, it could just be moved to src/config.ts and used by all specs.